src/authorization/authorization.guard.ts

```diff
@@ -6,7 +6,7 @@ import {
 } from '@nestjs/common';
 import { expressJwtSecret } from 'jwks-rsa';
 import { promisify } from 'util';
-import * as jwt from 'express-jwt';
+import { expressjwt, GetVerificationKey } from 'express-jwt';
```

Comment:
Why the need for GetVerificationKey? (I'm asking as a noob developer who wants to understand the code).

 I simply changed the import to:
const {expressjwt: jwt} = require("express-jwt") 
 and I left everything the same as it was done in the tutorial.